storage-local/src/main/java/io/kestra/storage/local/LocalStorage.java

```diff
@@ -57,6 +38,18 @@ public InputStream get(URI uri) throws IOException {
         );
     }
 
+    @Override
+    public List<URI> list(URI uri) throws IOException {
+        try (Stream<Path> stream = Files.list(getPath(URI.create(uri.getPath())))) {
+            return stream
+                .map(Path::getFileName)
+                .map(fileName -> URI.create("kestra://" +
+                    String.join("/", List.of(uri.getPath(), fileName.toString()))
+                ))
+                .toList();
+        }
+    }
```

Comment:
I'm a terrible reviewer sorry, just saw from my specs that it needs to return a List<[FILENAME, TYPE]> :'(